dashboard/test/ui/features/teacher_tools/course_overview.feature

```diff
@@ -1,3 +1,5 @@
+@skip
+# TODO: Reenable with new section setup flow
 Feature: CourseOverview
   Scenario: Viewing course overview signed out
```

Comment:
I'm also a big fan of leaving a jira ticket ID with any TODOs so that if you forget to do it someone else later can find it by ticket number (which you may have closed out or at least has more context).